### PR TITLE
增加連加連乘功能

### PR DESCRIPTION
**Pull request by XingZiLong**

### CODE REVIEW

**Comment by XingZiLong** [comment]
I use currying to provide this shortcut 😄😄😄

File: registry-packages.ts [hunk 1]
```diff
@@ -115,4 +115,14 @@ export const packages: PackageInfo[] = [
...
   },
+  {
+    name: '連加連乘',
+    repo: 'XingZiLong/continuous-operation',
+    description: 'Do continuous addition and mutiplication easily.',
+    author: {
+      name: 'XingZiLong',
+      url: 'httos://github.com/XingZiLong',
+    },
+    aliases: ['XingZiLong'],
```
Comment (antfu) [inline registry-packages.ts]: Aliases are for package name (normally it's pinyin but English is ok too). Looks like you mistakenly put the username here.

Comment (XingZiLong) [reply registry-packages.ts]: I have misunderstood it. Sorry, I will change it.

**Comment by XingZiLong** [comment]
How can I change the aliases ?

File: registry-packages.ts (hunk 1 above)
Comment (antfu) [inline registry-packages.ts]: Something like this, 
```suggestion
    aliases: ['lian-jia-lian-cheng'],
```
or (or both)
```suggestion
    aliases: ['continuous-operation'],
```
You can change it to anything you want as long as it does not conflict with other packages. By setting this, users can install your package by 
```
wyg install lian-jia-lian-cheng
```
or 
```
wyg install 連加連乘
```

Comment (XingZiLong) [reply registry-packages.ts]: Okay then, and am I finished?

Comment (XingZiLong) [reply registry-packages.ts]: I don’t see my package registry in the file registry-packages.ts

**Comment by antfu** [comment]
Cool. Thanks!

**Comment by XingZiLong** [comment]
Thanks!

**Pull request merged**